pkg/discovery/openapispec.go

```diff
@@ -0,0 +1,114 @@
+package discovery
```

Comment:
This file will serve other kinds of documentation as well besides OpenAPISpec for an Operator

So lets rename it to: operatordocs.go